ape_vyper/_utils.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Extension(Enum):
+    VY = ".vy"
+    VYI = ".vyi"
```

Comment:
might be a little bit more obvious this way
```suggestion
    SOURCE = ".vy"
    INTERFACE = ".vyi"
```

Reply:
Also if you add `__str__` method like I did in the silverback PR, you won't have to use `.value` everywhere

Reply:
What is wrong with using `.value` though? Like you could just do this:

```python
class MyEnum:
   VY = ".vy"
```

And skip using Enum altogether? I am still trying to see the benefit in `StrEnum`, please help me understand.

Reply:
```py
glob = f"*.{FileType.SOURCE}"  # Don't have to do `.value`, just calls `__str__`
```

Reply:
But then why use `Enum` at all? Because if you avoid the Enum base class, it works that way.

Reply:
like a class with just classvars, i guess then it would let you make instances which is kinda weird.
I suppose that is justification right there.

Reply:
note: still had to use .value in some places because of mypy / type-checking, which is think is fine.